stock_split_picking/wizards/stock_split_picking.py

```diff
@@ -43,6 +52,23 @@ def _apply_move(self):
                 new_pickings += picking._split_off_moves(move)
         return self._picking_action(new_pickings)
 
+    def _apply_available_line(self):
+        """Create different pickings for available and not available move line"""
+        for picking in self.mapped("picking_ids"):
+            if picking.picking_type_code == "outgoing":
```

Comment:
do not filter on "outgoing". The split can be used in other context like internal move.
For sure for incoming it do not make sense as incoming are always available, but I do not think we need to add extra code to block that case.